src/reversion/management/commands/createinitialrevisions.py

```diff
@@ -1,14 +1,22 @@
-from __future__ import unicode_literals, print_function
+from __future__ import unicode_literals
 
 from optparse import make_option
+try:
+    from collections import OrderedDict
+except ImportError:  # For Python 2.6
+    from django.utils.datastructures import SortedDict as OrderedDict
+
+try:
+    from django.apps import apps
+except ImportError:  # For Django < 1.7
+    from django.db import models as apps
```

Comment:
This feels like a fallback that isn't correct, as the two modules aren't equivalent. We should really be specifically importing get_models() and get_apps() using from django.(apps/models) import get_models, get_apps.